doc-content/enterprise-only/cases/case-management-case-file.con.adoc

```diff
@@ -22,3 +22,53 @@ You can specify any prefix. For example, if you specify the prefix `IT`, the fol
 `IT-0000000002`
 
 `IT-0000000003`
+
+You can also configure a variable as the business key, for example the following generates the `HR-TYPE1` prefix.
+
+[source]
+----
+{
+  case variable =>  type = "type1"
+  prefix = "HR-@{UPPER(type})"
+}
+----
+
+.Prefix HR-TYPE1
+
+[source]
+----
+{
+  HR-TYPE1-0000000001
+  HR-TYPE1-0000000002
+  HR-TYPE1-0000000003
+}
+----
+
+.Adding a different case variable, for example, ``type = 3type`
+
+[source]
+----
+{
+  HR-3TYPE-0000000001
+  HR-3TYPE-0000000002
+  HR-3TYPE-0000000003
+}
+----
+
+If the `IS_PREFIX_SEQUENCE` case variable is set to `false` (default value is `true`) it will not create any sequence and the `caseIdPrefix` expression evaluation is the caseId.
```

Comment:
This is not a case variable, this is case metadata 'CaseIdPrefixIsSequence'.

Reply:
Updated:

If the `CaseIdPrefixIsSequence` case metadata is set to `false` (default value is `true`) it will not create any sequence and the `caseIdPrefix` expression evaluation is the caseId.

Reply:
If the CaseIdPrefixIsSequence case metadata is set to false (default value is true) it does not create a sequence and the caseIdPrefix expression evaluation is the case ID. (or the value of the `caseID` variable).

Reply:
Done in Design doc. This topic removed for GSG.